### PR TITLE
Avoid shadowing global arch

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by nicolasbock** [comment]
Nice catch!

+1

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1066,10 +1066,11 @@ function onadmin_prepare_sles12_installmedia()
 
 function onadmin_prepare_sles12sp1_installmedia()
 {
-    for arch in aarch64 x86_64 s390x; do
-        local sles12sp1_mount="$tftpboot_suse12sp1_dir/$arch/install"
-        add_mount "SLE-12-SP1-Server-LATEST/sle-12-$arch" \
-            "$clouddata:/srv/nfs/suse-12.1/$arch/install" \
+    local a
```
Comment (nicolasbock) [inline scripts/qa_crowbarsetup.sh]: You could use `local arch` if you think the loop body is more readable that way.

Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: well $arch is a global variable, so I chose a different name to make it more obvious that we're referring tot he local name not the global name. I thought having a local and a global $arch would be even more confusing..

**Comment by mmnelemane** [comment]
+1

**Pull request merged**